app/presenters/tree_builder_utilization.rb

```diff
@@ -55,7 +55,7 @@ def x_get_tree_vandt_datacenter_kids(object)
         objects += rbac_filtered_sorted_objects(f.vms_and_templates, "name")
       elsif f.name == "host"            # Don't count host folder children
       else                              # add in other folders
-        objects += Rbac.filtered([f], :match_via_descendants => VmOrTemplate)
+        objects << Rbac.filter(f, :match_via_descendants => VmOrTemplate)
```

Comment:
f is a single object.
We are passing in an array of a single object
and then tacking those results onto an array (0 or 1 objects)

Looks like this may put a nil into there. I'll fix that. thanks